starboard/content/fonts/BUILD.gn

```diff
@@ -23,7 +23,6 @@ if (cobalt_font_package == "empty") {
   }
 } else {
   action("fonts_xml") {
-    install_content = true
```

Comment:
FYI @oxve we're removing the install_content bc install_targets aren't a thing right now